lib/Runtime/Language/JavascriptStackWalker.cpp

```diff
@@ -606,15 +698,15 @@ namespace Js
                 {
                     // Done walking inlined frames in a loop body, cache the native code address now
                     // in order to skip the loop body frame.
-                    this->SetCachedInternalFrameInfo(InternalFrameType_LoopBody, InternalFrameType_LoopBody);
+                    this->SetCachedInternalFrameInfo(InternalFrameType_LoopBody, InternalFrameType_LoopBody, true /*inlinedFramesOnStack*/);
                     isJavascriptFrame = false;
                 }
             }
 
             return true;
         }
 #endif
-
+        this->inlinedFramesOnStack = false;
```

Comment:
nit: you tend to alternate from adding is before a boolean to not doing it at all. I usually don't care about consistency issues like that, but you also do pointer compares in your conditional and assert statements. without the naming consistency  or equality checks in the conditionals\asserts it is hard to follow when something is a bool vs something else. I don't have the power of type inference over github so naming consistency would be very much appreciated.